src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateConverter.cs

```diff
@@ -346,6 +346,39 @@ internal static ContainerRegistryCredentials ConvertToContainerCredentials(Templ
             return result;
         }
 
+        internal static Snapshot ConvertToJobSnapshotRequest(TemplateContext context, TemplateToken token)
```

Comment:
are these changes copied from the service?
we we expect the golang parser handles this?

Reply:
I **_think_** the answer to these questions is similar to this comment: https://github.com/actions/runner/pull/3135/files#r1489319642

But please let me know if you need additional information. 

For context, we're unsure at this stage if we'll need expression support at some point and @ericsciple advised that it made sense to send the entire `TemplateToken` down to the runner and process it there instead of sending an evaluated `Snapshot` object. This was recommended as we would have the ability to check for expression tokens in the future. I'm happy to make changes if this wasn't the optimal approach though.